cookienotice/README

```diff
@@ -0,0 +1,7 @@
+Cookie Notice
+
+For server admins only.
+
+Configure, show and handle a simple cookie usage notice.
+
+Author: Peter liebetrau <https://socivitas.com/profile/peerteer>
```

Comment:
If you like, you can also make a Markdown formatted README.md file.

And TBH I would not really know what you mean with cookie usage notice, if I had not checked your profile.

Reply:
Ah great. Saw README.md with themes already. 
hmm .. the cookie usage notice  ... how else to call it ? ..

Reply:
I would describe it as absolute annoying but eventually necessary notification about the usage of cookies. This kind of things you klick ok on but don't read.

Reply:
Or, you could add a screen shot of such a message to the README.md file. Maybe this would spare a 1000 words description of the thing.

Reply:
done - used your perfect description

Reply:
didnt change to .md yet. Maybe with the next version